CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(ENABLE_GOOGLETEST_INSTALL "Skip installation" OFF )
```

Comment:
Final nitpick from my side:
`option(ENABLE_GOOGLETEST_INSTALL "Skip installation" ON)`
->
`option(ENABLE_GOOGLETEST_INSTALL "Install GoogleTest and GoogleMock" ON)`